bindings/python/tersets/__init__.py

```diff
@@ -77,19 +86,24 @@ def compress(values: List[float]) -> bytes:
 
     compressed_values = __CompressedValues()
 
-    configuration = __Configuration(0, 0.0)
-
+    if (type(method) != Method):
+        # Method does not exists, raise error, and show available options
```

Comment:
Missing punctuation at the end of the comment.

Reply:
Fixed.